man/unclutter-xfixes.man

```diff
@@ -32,6 +32,11 @@ rather just the desktop background.
 Ignore mouse scroll events (buttons 4 and 5) so that scrolling doesn't unhide
 the cursor.
 
+*--ignore-buttons*::
+Ignore mouse buttons clicks so that clicks doesn't unhide the cursor. Buttons
```

Comment:
Let's rephrase this a bit:

> Defines the mouse buttons which do not unhide the cursor when clicked. You can pass multiple button numbers by separating them with *\',\'*.

The rest should be removed. I don't think we need to add the complexity of supporting ranges, enumerating the buttons is easy enough to be acceptable. It's not like there are 200 buttons.

Maybe we can also add this to `--ignore-buttons`:

> This is a shortcut for *--ignore-buttons 4,5*.